src/clic.adoc

```diff
@@ -664,226 +608,79 @@ Bits   Field         Description
  29:28  mpp[1:0]      Previous privilege mode, mirror of mstatus.mpp
  27     mpie          Previous interrupt enable, mirror of mstatus.mpie
  26:24  (reserved)
- 23:16  mpil[7:0]     Previous interrupt level
+ 23:16  mpipprio[7:0] Previous interrupt preemption priority
  15:12  (reserved)
  11:0   exccode[11:0] Exception/interrupt code, mirror of mcause[11:0]
 ----
 
-Note: Switching to CLINT mode
-from CLIC mode causes {pil} in that privilege mode to be zeroed.
-
 ==== New Interrupt Status ({mintstatus}) CSR
 
-A new M-mode CSR, `mintstatus`, holds the active interrupt level for
+A new M-mode CSR, `mintstatus`, holds the active interrupt preemption priority for
 each supported privilege mode.  These fields are read-only.  The
 primary reason to expose these fields is to support debug.
 
 [source]
 ----
 mintstatus fields
  bits    description
- 31:24   mil
+ 31:24   mipprio
  23:16   (reserved)
- 15: 8   sil if ssclic is supported
+ 15: 8   sipprio if ssclic is supported
   7: 0   (reserved)
 ----
 
-==== New Interrupt-Level Threshold ({mintthresh}) CSRs

```

Comment:
Is this threshold functionality removed?
If so, how can software implement "semi-critical" sections where most interrupts are masked but high importance ones are let through?

Reply:
I removed this here, because i added a priority threshold to the Smclicincr extension in another PR.
Such a mechanism is needed with a high number of sources, independent of whether nested preemption is supported or not.